source/pmc/adding-committers.md

```diff
@@ -58,29 +98,164 @@ can do to become a better contributor. See also [Becoming A
+and then invite the candidate, using [this template](/templates/committer-results.txt) 
+for a new committer, or [this template](/templates/committer-pmc-results.txt) 
+for a committer and PMC member.
+
+We give candidates a chance to decline committership 
+in private. They can post a reply to the PMC mailing list.
+
+After we reach a decision on the `private@` list, and after the steps above, we
+[announce the new committer on the `dev` list](template/committer-announce.txt)
+
+Alternately, use [this template](/templates/committer-pmc-announce.txt) for new
+committer + PMC member.
+
+You can use [this template](/templates/committer-welcome.txt) to welcome
+the new committer to your project community. You are, however,
+encouraged to create your own version of this template, customized to
+your particular project community.
+
+Other notes about the process are available on the main 
+[Apache site](https://www.apache.org/dev/pmc.html#newcommitter).
+
+### Committer Account Creation
+Please see the [account creation instructions](https://www.apache.org/dev/pmc.html#newcommitter).
+
+In summary:
+
+If the ICLA identifies the project and a valid Apache ID, and the
+`[RESULT][VOTE]` message has been posted to the PMC private list,
+the account creation request is made by the
+secretary or assistant secretary who files the ICLA.
+
+Otherwise, the new account request should be made by the
+PMC Chair (or any [ASF Member](https://www.apache.org/foundation/glossary.html#Member)
+if the chair is unavailable).
 
-The complete policy and procedure around *how* to vote for a committer,
-and how to add them to the relevant rosters if and when they are voted
-in, is covered in the [PMC policy
-document](https://www.apache.org/dev/pmc.html#committer-management). As
-a PMC member, you should read and understand that document.
+The PMC chair needs to use the [ASF New Account Request](https://id.apache.org/acreq/pmc-chairs/) form to
+send a new account request. Members may use [ASF New Account Request](https://id.apache.org/acreq/members/) page.
 
-## What to do when a committer is elected
+Please supply the [mail archives](https://lists.apache.org/) URL as
+proof of the vote results.
 
-Once a committer has been added to the roster, be sure that they
-understand how things work in your project, rather than leaving them to
-figure this out on their own.
+<a name="NewCommitter-EmailTemplates"></a>
+## Email Templates
 
-Point them to specific review processes that they are expected to abide
-by before committing or merging a change.
+The following templates are recommended ways to phrase your email
+communications around inviting a new committer, to ensure that everyone
+understands your intent.
 
-Point new committers to [Infra's guide for new
-committers](https://infra.apache.org/new-committers-guide.html), which
-has general information for all ASF committers.
+* [Committer discussion template](/templates/committer-discuss.txt)
```

Comment:
Like this - using consistent link text throughout will help reinforce what specific links/documents mean.

Reply:
Done.